corehq/apps/products/views.py

```diff
@@ -256,7 +256,7 @@ def post(self, request, *args, **kwargs):
                 self.custom_data.is_valid()]):
             self.product.product_data = self.custom_data.get_data_to_save()
             self.new_product_form.save(self.product)
-            messages.success(request, _("Product saved!"))
+            messages.success(request, _("Product saved."))
```

Comment:
This will require this string to be re-translated, so I wouldn't change it here.

Reply:
fixed: https://github.com/dimagi/commcare-hq/pull/19574/commits/ef3121686456ffd8e920d6b4284058b74edf7681